src/content/blog/kai-roadmap-2024/index.mdx

```diff
@@ -0,0 +1,225 @@
+---
+author: jwmatthews
+date: 2024-07-26T07:00:00Z
+description: >-
+  Konveyor AI (Kai) Roadmap 2024
+featured: true
+image: header.png
+tags:
+  - Konveyor
+  - Kai
+  - GenAI
+  - AppModernization
+title: Konveyor AI (Kai) Roadmap 2024
+---
+
+**Author**: [John Matthews](https://github.com/jwmatthews)
+
+This post shares the roadmap for [Konveyor AI (Kai)](https://github.com/konveyor/kai) as of summer 2024.
+_For a recap of what is Konveyor AI (Kai) please see our prior post: [Kai - Generative AI Applied to Application Modernization](/blog/kai-deep-dive-2024/)_
+
+The roadmap is organized by themes of functionality, each focusing on a specific aspect of the project's development.
+
+Roadmap Outline:
+
+- [Purpose](#purpose)
+- [Maturity](#maturity)
+- [Guiding Principles](#guiding-principles)
+- [Themes](#themes)
+- [Milestones](#milestones)
+- [Future Areas to Consider](#future-areas-to-consider)
+- [Access to Early Builds](#access-to-early-builds)
+
+## Purpose
+**What is the purpose of Konveyor AI?**
+Kai intends to improve the economics of re-platforming and refactoring applications to Kubernetes and cloud-native technologies via use of Generative AI leveraging data in Konveyor.
+
+## Maturity
+**Maturity - Early Development - Pre Alpha**
+_Kai is in early stages of development and is NOT suitable for production usage at this time._
+
+- Please see [docs/Evaluation_Builds.md](docs/Evaluation_Builds.md) to learn more about early access preview builds.
```

Comment:
@jwmatthews this is with respect to Kai repo? We might need a fully qualified path here.

Reply:
I mean the hyperlink

Reply:
Thank you!